.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: groovy
+jdk:
+  - oraclejdk7
+script: ./grailsw refresh-dependencies
+  && ./grailsw "test-app unit:"
```

Comment:
I think you can specify this as a list too:
```
  - ./grailsw refresh-dependencies
  - ./grailsw "test-app unit:"
```
(feels slightly more readable to me -- YMMV).